stubs/networkx/networkx/algorithms/d_separation.pyi

```diff
@@ -1,7 +1,15 @@
+from binascii import Incomplete
```

Comment:
binascii? I think this recurs a few times

Reply:
ugh, bad Pylance autoimport I didn't notice. Thanks for catching it !